packages/docusaurus-1.x/lib/server/docs.js

```diff
@@ -61,19 +61,43 @@ function mdToHtmlify(oldContent, mdToHtml, metadata, siteConfig) {
     if (fencedBlock) return line;
 
     let modifiedLine = line;
+    const deb = (...args) => {
```

Comment:
Should this be here?

Reply:
Oh, I see you are still using it.  Instead of hard code, would an environment variable be appropriate?